components/config/histogram_config.py

```diff
@@ -18,6 +19,8 @@ def get_component(columns, min_date, max_date, default_end_date):
 
     return html.Div([
         date_picker.get_component(min_date, max_date, default_end_date),
-        histogram_column_picker.get_component(columns),
+        axis_picker.get_component(columns, default_column, is_histogram=True),
+        html.Div(className='label', children='Maximum number of bins'),
+        dbc.Input(id='nbins', type='number', min=2, step=1, value=10),
```

Comment:
This, as well as, `column` should be added to the list of IDs in `config.css` where the width of the input element is defined